application.py

```diff
@@ -1,13 +1,478 @@
-from flask import Flask, render_template
+import json
+import logging
+import os
+
+import boto3
+import requests
+from botocore.exceptions import ClientError
+from flask import (
+    Flask,
+    abort,
+    flash,
+    redirect,
+    render_template,
+    request,
+    session,
+    url_for,
+)
 from jinja2 import ChainableUndefined
 
+# Basic logging information
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
```

Comment:
We setup logger here but then use print a lot below, can we replace with logger.debug if you want to keep the print statements in.

Reply:
The print statements were left in for our testing. I've changed them to logging now.